### PR TITLE
fix docstring of SDDotVolumeTerm, SDDivTerm, SDDivGradTerm, SDConvect…

### PR DESCRIPTION
**Pull request by vlukes**
fix docstring of SDDotVolumeTerm, SDDivTerm, SDDivGradTerm, SDConvectTerm, SDGradDivStabilizationTerm, SDDiffusionTerm

### CODE REVIEW

File: sfepy/terms/terms_diffusion.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ class SDDiffusionTerm(Term):
         - material:    :math:`K_{ij}`
         - parameter_q: :math:`q`
         - parameter_p: :math:`p`
-        - parameter_v: :math:`\ul{\Vcal}`
+        - parameter_mesh_velocity: :math:`\ul{\Vcal}`
```
Comment (rc) [inline sfepy/terms/terms_diffusion.py]: Why this renaming? `parameter_v` seems to be correct according to `arg_types` below.

**Comment by rc** [comment]
Thanks, seems OK, except the remark above.

**Comment by vlukes** [comment]
The rest of sensitivity terms have "parameter_mesh_velocity" instead of "parameter_v".

**Comment by rc** [comment]
I see - then rename it everywhere in the class, please, so that the docstring is consistent with the code.

**Comment by vlukes** [comment]
I renamed it according to other sensitivity terms.

**Comment by rc** [comment]
Thanks1

**Pull request merged**